apps/web/src/components/UpcomingLessons/Content.tsx

```diff
@@ -68,6 +70,27 @@ export const Content: React.FC<{
     []
   );
 
+  const navigate = useNavigate();
+  const [members, setMembers] = useState<number[]>([]);
+  const [sendingMessage, setSendingMessage] = useState(0);
+
+  const findRoom = useFindRoomByMembers(members);
```

Comment:
This logic is the duplicated, please move it into a custom hook (to `web/src/hooks/chat.ts`) and re-use it in all places.